doc/nrf/getting_started.rst

```diff
@@ -20,6 +20,7 @@ After that, pick a sample that is related to the application you want to create
    gs_recommended_versions
    gs_assistant
    gs_installing
+   gs_updating
```

Comment:
So this is the change that actually creates a new page, right?

Reply:
Yes, this is a reference to the tag for the page I'm adding, so that it can be included and "created" in the hierarchy.